edge/registry/server.go

```diff
@@ -37,7 +38,7 @@ func NewEdgeRegistryHandler(edgeManager *edge.Manager, awsConfig *agent.AWSConfi
 func (handler *Handler) LookupHandler(rw http.ResponseWriter, r *http.Request) *httperror.HandlerError {
 	stackManager := handler.EdgeManager.GetStackManager()
 	if stackManager == nil {
-		return &httperror.HandlerError{http.StatusInternalServerError, "Unable to retrieve stack manager", errors.New("Stack manager is not available")}
+		return httperror.InternalServerError("Unable to retrieve stack manager", errors.New("Stack manager is not available"))
```

Comment:
please apply these changes to develop version

Reply:
Someone already made the changes to the develop branch before I could submit my pull request.